src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -322,9 +331,21 @@ class WelcomePage extends Disposable {
 	private onReady(container: HTMLElement, recentlyOpened: Promise<IRecentlyOpened>, installedExtensions: Promise<IExtensionStatus[]>): void {
 		const enabled = isWelcomePageEnabled(this.configurationService, this.contextService);
 		const showOnStartup = <HTMLInputElement>container.querySelector('#showOnStartup');
+		const wizardEnabled = isInitialSetupWizardEnabled(this.configurationService); // {{SQL CARBON EDIT}} - is the initial setup wizard enabled boolean
```

Comment:
why not group this change with the code block starting on line 339 to reduce number of places in `vs` code changed?